src/date-range-picker/__tests__/date-range-picker-absolute.test.tsx

```diff
@@ -241,6 +241,34 @@ describe('Date range picker', () => {
       });
     });
 
+    describe('time formats', () => {
+      test('correctly handles a time format of hh', () => {
+        const onChangeSpy = jest.fn();
+        const { wrapper } = renderDateRangePicker({
+          ...defaultProps,
+          timeInputFormat: 'hh',
+          onChange: event => onChangeSpy(event.detail),
+        });
+
+        act(() => wrapper.findTrigger().click());
```

Comment:
Test utils already call `act` when needed. No need to do it one more time

Reply:
Removed from my tests. The entire file wraps everything in act though. I can do a follow up PR to remove it from the rest as well.

Reply:
Yes we have a lot of old code, pre-dating this change: https://github.com/cloudscape-design/test-utils/pull/25

Any cleanups would be nice, easy wins there